NetCoreRestApi/DataLayer.EF/ProductsDbContext.cs

```diff
@@ -0,0 +1,107 @@
+﻿using DataLayer.EF.Converters;
+using DataLayer.EF.Entities;
+using DataLayer.Models;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+
+namespace DataLayer.EF
+{
+    public class ProductsDbContext : DbContext, IDbContext       
+    {
+        public ProductsDbContext(DbContextOptions<ProductsDbContext> options) : base(options)
+        {
+            AddRange(GetProductsMock());
```

Comment:
This code shouldn't be here as it is mock data. So create ProductsMockDbContext that inherits ProductDbContext and move seeding code into that class and configure DI to use this class instead of ProductDbContext

Reply:
fixed